libs/test-utils/src/mocks/orml_asset_registry.rs

```diff
@@ -10,26 +10,33 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
+pub mod reexport {
+	pub use staging_xcm::{
+		latest::prelude::MultiLocation as __private_MultiLocation,
```

Comment:
Nit - maybe we can use the actual version here instead of `latest` which I assume it's `v3` now?

Reply:
Yeah, it could be 👍🏻. I will do this in the next commit with more possible NITs.

Anyway, I think we should remove that mock soon. Each time something changes in the asset registry and you need to update the code into that macro, it is quite complicated to make it work again.